online-shop/src/app/mocks/products.mocks.ts

```diff
@@ -0,0 +1,20 @@
+import { Product } from "../modules/shared/types/products.types";
+
+export const MOCK_PRODUCTS: Product[] = [
+    {
+        "id": "7ed8b2fa-0d7f-4dab-a01e-5583841e0469",
+        "name": "Notebook Basic 15",
+        "category": "Laptops",
+        "image": "https://sapui5.hana.ondemand.com/test-resources/sap/ui/documentation/sdk/images/HT-1000.jpg",
+        "price": 956,
+        "description": "Notebook Basic 15 with 2,80 GHz quad core, 15\" LCD, 4 GB DDR3 RAM, 500 GB Hard Disc, Windows 8 Pro"
+    } as Product,
+    {
+        "id": "6a5865a4-1dfd-46cf-8958-2a8950d5789a",
```

Comment:
Also, there is no need to add " for the properties of the product. That's the nice thing about Typescript <3 The first one from the screenshot will be the correct version. You can reach a conclusion by looking at the second one: (and notice no compile errors)
![image](https://github.com/MarkDMsg/AngularTraining/assets/33027937/0df5ebdc-ba9d-474b-8bcf-1ca5c74be9b2)